test/mocks/mappings/questions-error.json

```diff
@@ -17,17 +17,28 @@
       "newScenarioState": "FirstQuestion",
       "request": {
         "method": "POST",
-        "url": "/authorize",
```

Comment:
I wonder why it was hitting `authorize` beforehand - but this change makes sense.